docs/themes/oso-tailwind/assets/css/site.css

```diff
@@ -10,4 +8,171 @@
     c700: #211164;
     c700: #170950;
     c900: #0e024e;
-} */
\ No newline at end of file
+} */
+
+.search-button {
+    display: flex;
+    -webkit-box-align: center;
+    align-items: center;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    width: 100%;
+    border-radius: 0.25rem;
+    border-width: 1px;
+    border-color: #D6D8DC;
+    cursor: pointer;
+    padding: 0.5rem;
+    color: #B1B6BE;
+    text-align: left;
+}
+
+@media (min-width: 768px) {
```

Comment:
use tailwind for media queries if possible in case we reconfigure our size breaks

that way they will still match